Source/JavaScriptCore/builtins/PromiseConstructor.js

```diff
@@ -136,7 +136,7 @@ function all(iterable)
     var promise = @newPromise();
     var values = [];
     var globalContext = {
-        promise,
+        @promise: promise,
```

Comment:
Let's not use private symbols. This object is guaranteed not to be exposed to users. Thus just use normal properties.